src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -1771,6 +1772,8 @@ private void ExecuteSearchPackageCommand(object sender, ExecutedRoutedEventArgs
 
         private async Task ExecuteRestartSearchCommandAsync()
         {
+            // reset vulnerability data
```

Comment:
Remove comment. The next line is self-explanatory and the comment does not provide any extra context.